crates/srs/src/voice_stream.rs

```diff
@@ -26,6 +26,7 @@ use tokio::time;
 use tokio_util::codec::{FramedRead, FramedWrite};
 use tokio_util::udp::UdpFramed;
 
+use std::default::Default;
```

Comment:
Could you move this import to the `std` import block at the beginning of the file? On that note, why is this import even necessary, I think `Default` is available by default (without an import)?